libraries/DateTime/test/dateTime.js

```diff
@@ -0,0 +1,344 @@
+    })
+
+    it('get_minute', async () => {
+      const array = [
+        { minute: 59, timestamp: 63071999 },
+        { minute: 4, timestamp: 63072240 },
+        { minute: 10, timestamp: 63072600 },
+        { minute: 18, timestamp: 63073080 },
+        { minute: 25, timestamp: 63073559 },
+      ]
+      async function withdrawMinute(item, timestamp) {
+        const result = await callContract(
+          contract,
+          'get_minute',
+          timestamp
+        ).then(async item => item.decode('int'))
+        return result
+      }
+      const decodedMinutes = await processArray(array, withdrawMinute)
+      const mockupMinutes = array.map(item => item.minute)
+
+      assert.deepEqual(decodedMinutes, mockupMinutes)
+    })
+
+    it('get_second', async () => {
+      const array = [
+        { second: 59, timestamp: 63071999 },
+        { second: 6, timestamp: 63072006 },
+        { second: 17, timestamp: 63072017 },
+        { second: 23, timestamp: 63072023 },
+        { second: 28, timestamp: 63072028 },
+      ]
+      async function withdrawSecond(item, timestamp) {
+        const result = await callContract(
+          contract,
+          'get_second',
+          timestamp
+        ).then(async item => item.decode('int'))
+        return result
+      }
+      const decodedSeconds = await processArray(array, withdrawSecond)
+      const mockupSeconds = array.map(item => item.second)
+
+      assert.deepEqual(decodedSeconds, mockupSeconds)
+    })
+
+    it('get weekday', async () => {
+      const array = [
+        { weekday: 3, timestamp: 67737599 },
+        { weekday: 5, timestamp: 67824000 },
+        { weekday: 0, timestamp: 68083199 },
+        { weekday: 2, timestamp: 68169600 },
+        { weekday: 4, timestamp: 68342400 },
+      ]
+      async function withdrawWeekday(item, timestamp) {
+        const result = await callContract(
+          contract,
+          'get_weekday',
+          timestamp
+        ).then(async item => item.decode('int'))
+        return result
+      }
+      const decodedWeekdays = await processArray(array, withdrawWeekday)
+      const mockupWeekdays = array.map(item => item.weekday)
+
+      assert.deepEqual(decodedWeekdays, mockupWeekdays)
+    })
+  })
+
+  describe('Test to timestamp conversion', () => {
+    before(async () => {
+      contract = await deployContract(owner)
+    })
+    it('to_timestamp function', async () => {
+      const year = 2006
+      const month = 7
+      const day = 4
+      const hour = 9
+      const minute = 43
+      const second = 7
+      const timestamp = 1152006187
+
+      const callWithdraw = await contract
+        .call('to_timestamp', {
+          args: `(${year}, ${month}, ${day}, ${hour}, ${minute}, ${second})`,
+          options: { ttl: config.ttl },
+          abi: 'sophia',
+        })
+        .then(async item => item.decode('int'))
+      assert.equal(callWithdraw.value, timestamp)
+    })
```

Comment:
you may add and this test ...
// let date =  new Date(2016, 1, 29, 10, 45, 32); // or new Date() 
      let date =  new Date() 
      const timestamp = date.getTime();

      const year = date.getUTCFullYear();
      const month = date.getUTCMonth() + 1;
      const day = date.getUTCDate();
      const hour = date.getUTCHours();
      const minute = date.getUTCMinutes();
      const second = date.getUTCSeconds();

      let aeTimestamp = parseInt(timestamp / 1000); // to AE_timestamp
                      
      const callWithdraw = await contract
        .call('to_timestamp', {
          args: `(${year}, ${month}, ${day}, ${hour}, ${minute}, ${second})`,
          options: { ttl: config.ttl },
          abi: 'sophia',
        })
        .then(async item => item.decode('int'))
      assert.equal(callWithdraw.value, aeTimestamp)